app/tests/spec/lib/fxa-client.js

```diff
@@ -1581,11 +1581,11 @@ define(function (require, exports, module) {
         };
         sinon.stub(realClient, 'verifyTotpCode').callsFake(() => Promise.resolve(resp));
 
-        return client.verifyTotpCode('code')
+        return client.verifyTotpCode('code', relier)
```

Comment:
It seems like this test should be added to the Account model instead of here. [fxa-client->verifyTotpCode](https://github.com/mozilla/fxa-content-server/blob/70e6a578a86a650f57cbad61c77ae36c5d7ba3c5/app/scripts/lib/fxa-client.js#L1011) takes an `options` object as its second parameter rather than a relier.